<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>632</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>581</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>login</string>
     </attribute>
     <widget class="QPushButton" name="login_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter/Modifier</string>
     </attribute>
     <widget class="QLineEdit" name="id_emp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="salaire">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_emp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>salaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date_entre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLineEdit" name="id_empSupp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_emp</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tablemp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="trier">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id_emp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>salaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date_entre</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string> pdf</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>rechercher</string>
     </attribute>
     <widget class="QPushButton" name="recherche_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QComboBox" name="recherche_type">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id_emp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date_entre</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="text_recherche">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tablemp_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>401</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="time_label_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>mailing</string>
     </attribute>
     <widget class="QPushButton" name="sendmail">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>sendmail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwrd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="object">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>object</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recep">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>email adresse</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>236</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="time_label_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_ajouter_emp">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label_id_c">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>CODE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cin_c">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_c">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenom_c">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_code">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter_E">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_quitter">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_E">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rafraichir_E">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rafraichir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ville</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ville">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-80</y>
        <width>511</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Evenement</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_afficher_emp">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer_E">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rafraichir">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rafraichir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2_quitter">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tri_codee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri code</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tri_nom">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tri_prix">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercher_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercher_ville">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chercher_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher type</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chercher_ville">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher ville</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>621</width>
        <height>261</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_statistique">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher Statistique</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gestion des employes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gestion des evenements</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
